+++
title = "Substructured population growth in the Ashkenazi Jews inferred with Approximate Bayesian Computation"

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date = 2018-11-04T14:50:13-04:00
date_end = 2018-11-07T14:50:13-04:00
all_day = true

# Schedule page publish date (NOT talk date).
publishDate = 2019-04-27T14:50:13-04:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Ariella Gladstein", "Michael Hammer"]

# Location of event.
location = "Cold Spring Harbor Laboratory, New York"

# Name of event and optional event URL.
event = "Probabilistic Modeling In Genomics"
event_url = "https://meetings.cshl.edu/meetings.aspx?meet=PROBGEN&year=18"

# Abstract. What's your talk about?
# abstract = "The Ashkenazi Jews (AJ) are a population isolate that have resided in Central Europe since at least the 10th century and share ancestry with both European and Middle Eastern populations. Relative isolation and large growth following a bottleneck led to a high frequency of founder mutations associated with disease. Between the 11th and 16th centuries, AJ expanded eastward leading to two culturally distinct communities, one in central Europe and one in eastern Europe. Different frequencies of mutations associated with diseases have been found among AJ communities, however most medical studies treat the AJ as one isolated population. Our aim was to determine if there are genetically distinct AJ subpopulations that reflect these cultural groups, and if so, what demographic events contributed to the population differentiation. We used Approximate Bayesian Computation (ABC) to choose among models of AJ history and infer demographic parameter values, including divergence times, effective population sizes, and migration rates. For the ABC analysis we used allele frequency spectrum and identity by descent based statistics to capture genealogical information across a wide timescale. We also mitigated the effects of ascertainment bias when performing ABC on SNP array data by modeling the SNP discovery process that created the SNP array. We found that the most likely model included a split between the Eastern and Western AJ subpopulations ~400 years ago. The differentiation between the Eastern and Western AJ could be attributed to more extreme population growth in the Eastern AJ (0.25 per generation) than the Western AJ (0.069 per generation)."

# Summary. An optional shortened abstract.
summary = ""

# Is this a featured talk? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Markdown Slides (optional).
#   Associate this page with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Optional filename of your slides within your talk folder or a URL.
url_slides = ""

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Links (optional).
url_pdf = "talk/ashkenazi/AJ_poster_long.pdf"
url_video = ""
url_code = "https://github.com/agladstein/AJ_ABC"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Right"
+++
